OBUSPA: Removed hardcoded CFLAGS in configure.ac
  
The hardcoded CFLAGS was inadvertently removing the include path
for the CCSP API header.
This patch has been submitted upstream.


Date: Fri, 19 Feb 2021 00:00:00 +0000
From: Richard Holme <richard.holme@commscope.com>
Subject: [PATch] Remove hardcoded C flags
Source: CommScope
License: BSD-3-Clause
Upstream-Status: Pending
Signed-off-by: Richard Holme <richard.holme@commscope.com>


diff -rupN old/configure.ac new/configure.ac
--- old/configure.ac	2020-09-29 16:40:39.000000000 +0100
+++ new/configure.ac	2020-10-23 12:35:55.511646053 +0100
@@ -12,7 +12,6 @@ LT_PREREQ([2.2])
 LT_INIT([shared disable-static])
 
 # Checks for programs.
-CFLAGS=" -g -O0"
 AC_PROG_CC
 AM_PROG_CC_C_O
 
